src/inquirer/render/console/_checkbox.py

```diff
@@ -10,9 +10,15 @@
 class Checkbox(BaseConsoleRender):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.selection = [k for (k, v) in enumerate(self.question.choices) if v in (self.question.default or [])]
+        self.locked = self.question.locked or []
+        self.selection = [k for (k, v) in enumerate(self.question.choices) if v in (self.set_default_choices())]
         self.current = 0
 
+    def set_default_choices(self):
+        default = self.question.default or []
+        final = set(default + self.locked)
+        return list(final)
```

Comment:
why do you cast into a set and than back into a list? This semse redundant and bad-practise.

`in` works just as well on a set or a list with doubles in it..

Reply:
fixed